docs/network/smart-rollup-nodes.md

```diff
@@ -278,6 +278,21 @@ Follow these steps to convert a Smart Rollup node from observer mode to maintena
      --pre-images-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0
    ```
 
+   If you are using a remote signer, pass the address of the remote signer in the `--remote-signer` argument before the `run` command, as in this example:
+
```

Comment:
Maybe we could add the command that needs to be used for octez client :

```bash
octez-client \
  --remote-signer tcp://localhost:7732/ \
  import secret key <REMOTE_OPERATOR> remote:tz1QCVQinE8iVj1H2fckqx6oiM85CNJSK9Sx
```

Reply:
That's covered earlier in the section. Is this what you mean?
https://github.com/etherlinkcom/docs/pull/271/files#diff-ae8151eddca9236c113704d4f5d508940f28597e1e02c6f31f918f925299ba6fL219-L237

Reply:
If we import with the last command we won't be able to use --remote-signer because the address is hardcoded.

Reply:
Got it, added `octez-client --remote-signer tcp://localhost:7732/ import secret key <REMOTE_OPERATOR> remote:tz1QCVQinE8iVj1H2fckqx6oiM85CNJSK9Sx`.

Reply:
Looks good, thanks